flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBConfigurableOptions.java

```diff
@@ -64,6 +71,20 @@
                             "The maximum number of open files (per TaskManager) that can be used by the DB, '-1' means no limit. "
                                     + "RocksDB has default configuration as '-1'.");
 
+    public static final ConfigOption<InfoLogLevel> LOG_LEVEL =
+            key("state.backend.rocksdb.log.level")
+                    .enumType(InfoLogLevel.class)
+                    .noDefaultValue()
+                    .withDescription(String.format(
+                            "The specified log level for DB. Candidate log level is %s, %s, %s, %s, %s or %s, "
+                                    + "and Flink choose '%s' as default style. "
```

Comment:
I think we could remove this line as it add description of `so Flink sets the RocksDB log level to HEADER_LEVEL by default.`

Reply:
@Myasuka Thanks for your review. I have removed.